leetcode/581.shortest-unsorted-continuous-subarray.py

```diff
@@ -5,24 +5,24 @@ def findUnsortedSubarray(self, nums):
         :rtype: int
         """
         n = len(nums)
-        l, i = 0, 0
+        l, i = n-1, 0
         for i in range(1, n):
             if nums[i]<nums[i-1]:
-                l = i
+                l = i-1
                 break
 
-        for i in range(i, n):
-            while l>=0 and nums[i]<nums[l]:
+        for i in range(i+1, n):
```

Comment:
```
Input: [1,3,2,2,2]
Output: 2
Expected: 4
```
should be `i`, not `i+1`. as `i` is the first reversed element, need to extend to find left boundary